src/background/adblocker.js

```diff
@@ -23,11 +24,18 @@ import * as trackerdb from '/utils/trackerdb.js';
 import * as OptionsObserver from '/utils/options-observer.js';
 import Request from '/utils/request.js';
 import asyncSetup from '/utils/setup.js';
-import { debugMode } from '/utils/debug.js';
 
 import { tabStats, updateTabStats } from './stats.js';
 import { getException } from './exceptions.js';
 
+const scriptlets = new Map();
+for (const [name, scriptlet] of Object.entries(SCRIPTLETS)) {
```

Comment:
I would move this to the `@ghostery/scriptlets` as default export (a final map).

Reply:
done - please review related PR https://github.com/ghostery/scriptlets/pull/1